Jenkinsfile

```diff
@@ -179,21 +181,20 @@ pipeline {
                         sh '''rm -rf artifacts/ubuntu18.04/
                               mkdir -p artifacts/ubuntu18.04/
                               mkdir -p _topdir
+                              rm -rf *.tar.gz
                               rm -rf _topdir/BUILD/
-                              rm -rf _topdir/SOURCES
                               mkdir -p _topdir/BUILD
-                              mkdir -p _topdir/SOURCES
-                              : "${DEBEMAIL:="nomail@hpdd.intel.com"}"
-                              : "${DEBFULLNAME:="Internal HPDD Devops"}"
+                              : "${DEBEMAIL:="daos@daos.groups.io"}"
+                              : "${DEBFULLNAME:="daos-stack"}"
                               : "${LANG:="C.UTF-8"}"
                               : "${LC_ALL:="C.UTF-8"}"
                               export DEBEMAIL
                               export DEBFULLNAME
                               export LANG
                               export LC_ALL
                               if make debs; then
```

Comment:
We are actually not taking full advantage of Pipeline's capabilities by using this kind of construct, and our Jenkinsfile is less idiomatic because of it.  I realise I started doing it this way but I have been migrating toward this: 
https://github.com/daos-stack/openpa/blob/754bc9929b99a5065d8789d8e99652fb0f137153/Jenkinsfile#L83-L108
which as you can see, takes advantage of Jenkins' Pipeline to remove the success/failure handling code from the `sh` step and putting it into `post{}`.  This makes the success/failure code immune to errors in the handling of the main step code and ensures they are always run no matter why kind of unknown syntax error that might exist in the main steps.

Reply:
Fixed